### PR TITLE
Change git -C to git --git-dir

### PR DESCRIPTION
**Pull request by woyczek**
Fix defect #19

### CODE REVIEW

File: src/Main.hx [hunk 1]
```diff
@@ -48,8 +48,8 @@ class Main {
 
 	static function fetch(remote, local, callback)
 	{
-		ChildProcess.exec('git -C "$local" remote set-url origin "$remote"', function(err, stdout, stderr) {
-			ChildProcess.exec('git -C "$local" fetch --quiet', callback);
+		ChildProcess.exec('git -git-dir "$local" remote set-url origin "$remote"', function(err, stdout, stderr) {
+			ChildProcess.exec('git -git-dir "$local" fetch --quiet', callback);
```
Comment (jonasmalacofilho) [inline src/Main.hx:52]: Both are missing a dash here (i.e. `--git-dir`, not `-git-dir`).

**Review by jonasmalacofilho — CHANGES_REQUESTED** [review]
What version of `git` do you have?

I can't reproduce your issue on 2.26.0, and I don't see anything on the changelog for 2.27.0 suggesting the `-C` options has been removed.

**Comment by jonasmalacofilho** [comment]
Oh, sorry, the git version was in the title of your issue... 1.8.3.1.  As it turns out `git -C` was only [introduced](https://github.com/git/git/commit/44e1e4d67d5148c245db362cc48c3cc6c2ec82ca) in 1.8.5.  

Any chance you can update to a newer version?

If you can't (please let me know why), I don't think I see any problem in merging this (after you fix the minor problem I noticed).

**Comment by jonasmalacofilho** [comment]
Ping @woyczek.

**Comment by woyczek** [comment]
﻿No, absolutely no chance, since it is a bulk product from packaged RHEL

— 
This machine has no brain.
Use your own.

Gaétan Ryckeboer

**Pull request closed without merging**